conf/local.conf.append

```diff
@@ -0,0 +1,2 @@
+# Uncomment to enable tracing
+#USER_FEATURES += "tracing"
```

Comment:
this will be duplicate of this https://github.com/MentorEmbedded/meta-mentor-bsp/blob/master/common/conf/local.conf.append

Reply:
Updated.

Reply:
@Noor-Ahsan Removed it 2 days ago.